motionplan/tpSpaceRRT.go

```diff
@@ -465,8 +465,15 @@ func (mp *tpSpaceRRTMotionPlanner) getExtensionCandidate(
 	var targetFunc ik.StateMetric
 	relPose := spatialmath.PoseBetween(nearest.Pose(), randPosNode.Pose())
 	targetFunc = mp.algOpts.goalMetricConstructor(relPose)
+	seedDist := relPose.Point().Norm()
+	seed := tpspace.PTGIKSeed(curPtg)
+	dof := curPtg.DoF()
+	if seedDist < dof[1].Max {
```

Comment:
We're repeating ourselves a little with this code block which is repeated on line 810.  Can we change the PTGIKSeed function to operate on start and goal poses or something similar?  This is pretty minor though and I don't feel strongly if you'd rather preserve the function signature as it currently exists

Reply:
This will go away imminently with Nick S's work on RSDK-6052, which is why I felt okay duplicating this

Reply:
So my inclination is to let it be